tests/main/nfs-support/task.yaml

```diff
@@ -0,0 +1,136 @@
+summary: Test that snaps still work when /home is a NFS mount
+details: |
+    Snapd now contains a feature where NFS-mounted /home (or any sub-directory)
+    initializes a workaround mode where all snaps gain minimal amount of network
+    permissions sufficient for NFS to operate.
+systems: [ubuntu-16.04-64]  # TODO: expand this list
+prepare: |
+    . "$TESTSLIB/pkgdb.sh"
+    . "$TESTSLIB/snaps.sh"
+
+    # Install NFS server and a simple shell snap.
+    distro_install_package nfs-kernel-server
+    distro_install_package bsdgames
```

Comment:
The package installations should happen at the global location next to every other package installation, and it doesn't have to be purged at restore time. Please coordinate with Sergio so that these packages also go into the image maintenance spread runs so that once we finally manage to cook packages into the images this will be there too.

Reply:
I'm coordinating this with Sergio now.